commands.js

```diff
@@ -2149,6 +2149,27 @@ let commands = exports.commands = {
 	},
 	unblacklisthelp: ["/unblacklist [username] - Unblacklists the user from the room you are in. Requires: # & ~"],
 
+	unblacklistall: function (target, room, user) {
+		if (!this.can('editroom', null, room)) return false;
+
+		if (!target) {
+			user.lastCommand = '/unblacklistall';
+			this.errorReply("THIS WILL UNBLACKLIST ALL BLACKLISTED USERS IN THIS ROOM.");
+			this.errorReply("To confirm, use: /unblacklistall confirm");
+			return;
+		}
+		if (user.lastCommand !== '/unblacklistall' || target !== 'confirm') {
+			return this.parse('/help unblacklistall');
+		}
+
+		let unblacklisted = Punishments.roomUnblacklistAll(room);
+		if (!unblacklisted) return this.errorReply("No users are currently blacklisted in this room to unblacklist.");
+		let reply = `All blacklists in this room have been lifted by ${user.name}.`;
+		room.add(`|c|${user.getIdentity()}|/log ${reply}`).update();
+		this.logEntry(`${reply}  This has unblacklisted: ${unblacklisted.join(', ')}`);
```

Comment:
Funny you mention that actually, I was taught to always put two spaces in-between sentences in school when I was learning how to type.  I'll remove this though, now that I'm remembering this isn't a standard for PS stuff.